### PR TITLE
Pareto/NBD Example Notebook

### PR DESCRIPTION
**Pull request by ColtAllen**
## Description

This PR adds an example notebook for `ParetoNBDModel` to the documentation, along with a minor change in the importing convention for the utility functions (ie, they can now be imported from either `clv` or `clv.utils`).

 This PR took longer than anticipated because the notebook itself is rather in-depth. I decided against adding more plots or a section on future model enhancements, but if anyone feels like the notebook length isn't overlong and would like to see this additional content, please let me know.

## Related Issue

- [ ] Closes # https://github.com/pymc-labs/pymc-marketing/issues/524
- [ ] Related to # https://github.com/pymc-labs/pymc-marketing/issues/95

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [x] Added necessary documentation (docstrings and/or example notebooks)
- [x] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Modules affected

- [ ] MMM
- [x] CLV

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc-marketing--646.org.readthedocs.build/en/646/

### CODE REVIEW

**Comment by juanitorduz** [comment]
@ColtAllen This is pure GOLD! It is fantastic! I have a little request about adding some x/y labels and that's it ✅ 

Regarding:

>  I decided against adding more plots or a section on future model enhancements, but if anyone feels like the notebook length isn't overlong and would like to see this additional content, please let me know.

Everything regarding examples and docs is welcome! Therefore, it would be great to include these other plots you have in mind. Still, I leave it up to you if you want to do this in this PR or in a future one (please create an issue so that we keep track of it if you decide to do it in a different PR)

**Comment by ColtAllen** [comment]
> @ColtAllen This is pure GOLD! 🚀 🎸
> 
> It is fantastic! I have a little request about adding some x/y labels and that's it ✅
> 
> Regarding:
> 
> > I decided against adding more plots or a section on future model enhancements, but if anyone feels like the notebook length isn't overlong and would like to see this additional content, please let me know.
> 
> Everything regarding examples and docs is welcome! Therefore, it would be great to include these other plots you have in mind. Still, I leave it up to you if you want to do this in this PR or in a future one (please create an issue so that we keep track of it if you decide to do it in a different PR)

Thanks! x/y labels and legends have been added to the relevant plots, and I was considering adding another heatmap showing how different covariate values impact the average number of purchases for a new customer, but it seemed kinda extra.

The plots also seemed to be stretched out in the Docs preview.

**Review by juanitorduz — APPROVED** [review]

**Comment by juanitorduz** [comment]
@ColtAllen, thank you for this great addition! You can update the PR and once is green, feel free to merge :) 

(I can also do it if you want)

**Pull request merged**
